Formula/slirp4netns.rb

```diff
@@ -0,0 +1,31 @@
+class Slirp4netns < Formula
+  desc "User-mode networking for unprivileged network namespaces"
+  homepage "https://github.com/rootless-containers/slirp4netns"
+  url "https://github.com/rootless-containers/slirp4netns/archive/refs/tags/v1.1.11.tar.gz"
+  sha256 "87a8909746781d995b1b49eb36540e6ee745599f983c18f9b4e927ec92d86eb6"
+  license "GPL-2.0-only"
```

Comment:
```suggestion
  license "GPL-2.0-or-later"
```
From the SPDX identifier header found in `main.c`.